src/lidar/MainVis.cpp

```diff
@@ -70,8 +77,7 @@ int runRPLidar(unsigned long baudrate) {
 			double dtheta = (scan.value().angle_max-scan.value().angle_min)/(scan.value().ranges.size()-1);
 			for (long unsigned i = 0; i < scan.value().ranges.size(); i++) {
 				double rad = dtheta*i;
-				double MM_TO_M = 1000;
-				double dist = scan.value().ranges[i] * MM_TO_M;
+				double dist = scan.value().ranges[i] * Constants::Lidar::MM_PER_M;
```

Comment:
It looks like `scan.value().ranges[i]` is in meters, so no conversions necessary?